### DISCUSSION TITLE
SpanProcessor in Java SDK autoconfiguration

### DISCUSSION THREAD

**Discussion by esaabdk** [open-telemetry/opentelemetry-java-instrumentation#10392] [Q&A]
Does Java SDK autoconfiguration uses BatchSpanProcessor in the java agent?

Couldn't find clearly from here: https://opentelemetry.io/docs/languages/java/automatic/agent-config/

Kindly clarify.

Thanks.

**Comment by jkwatson** [comment]
moving this to the instrumentation repo where the agent lives

**Comment by trask** [comment]
> Does Java SDK autoconfiguration uses BatchSpanProcessor in the java agent?

yes, it does

> Couldn't find clearly from here: https://opentelemetry.io/docs/languages/java/automatic/agent-config/

check out the link there to [Batch span processor](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#batch-span-processor)
